bot/src/stake_pool_v0.rs

```diff
@@ -930,7 +930,7 @@ mod test {
         let (test_validator, authorized_staker) = test_validator_genesis.start();
 
         let websocket_url = &test_validator.rpc_pubsub_url();
-        let (rpc_client, _recent_blockhash, _fee_calculator) = test_validator.rpc_client();
```

Comment:
(commenting here because GitHub doesn't let you comment on unchanged lines)

Test fails with this:

```
---- stake_pool_v0::test::this_test_is_too_big_and_slow stdout ----
Waiting for fees to stabilize 1...
Waiting for fees to stabilize 2...
thread 'stake_pool_v0::test::this_test_is_too_big_and_slow' panicked at 'called `Result::unwrap()` on an `Err` value: RpcError(ClientError { request: Some(GetSlotLeaders), kind: RpcError(RpcResponseError { code: -32602, message: "Invalid slot range: leader schedule for epoch 3 is unavailable", data: Empty }) })', bot/src/stake_pool_v0.rs:882:14
```

The function that is returning the Err is `uniform_stake_pool_apply`, though it seems like it only fails on the _third_ time it is called (first it applies None to all validators, then it applies Baseline to all validators, then it fails on applying Bonus to all validators.